metrics.go

```diff
@@ -5,6 +5,17 @@ import (
 )
 
 var (
+	// Size buckets from 256 KiB (default chunk in Kubo) to 4MiB (maxBlockSize), 256 KiB  wide each
+	blockSizeHistogram = prometheus.LinearBuckets(262144, 262144, 16)
+
+	// TODO: Speed max bucket could use some further refinement,
+	// for now we don't expect speed being  bigger than transfering 4MiB (max block) in 500ms
+	speedHistogram = prometheus.ExponentialBucketsRange(1, 4194304/500, 20)
+
+	// Duration max bucket is informed by the timeouts per block and per peer request/retry
+	durationPerBlockHistogram        = prometheus.ExponentialBucketsRange(1, 60000, 10)
+	durationPerBlockPerPeerHistogram = prometheus.ExponentialBucketsRange(1, 20000, 10)
```

Comment:
:information_source:  I've just remembered that it is a good practice to avoid defining buckets with values that may
change. By hardcoding bucket definitions this way, we avoid breaking grafana boards when someone decides to adjust a timeout.

If we ever need to change buckets, or add new one with bigger max, but keep old ones intact, we can define them as explicit list of values (like we did with legacy ones in kubo a while ago):

```go
[]float64{0.05, 0.1, 0.25, 0.5, 1, 2, 5, 10, 30, 60}
```